dspace-api/src/test/java/org/dspace/app/packager/PackagerIT.java

```diff
@@ -0,0 +1,226 @@
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ *
+ * http://www.dspace.org/license/
+ */
+package org.dspace.app.packager;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.Iterator;
+import java.util.UUID;
+import java.util.zip.ZipEntry;
+import java.util.zip.ZipFile;
+
+import com.google.common.collect.Iterators;
+import org.dspace.AbstractIntegrationTestWithDatabase;
+import org.dspace.builder.CollectionBuilder;
+import org.dspace.builder.CommunityBuilder;
+import org.dspace.builder.ItemBuilder;
+import org.dspace.content.Collection;
+import org.dspace.content.Community;
+import org.dspace.content.Item;
+import org.dspace.content.WorkspaceItem;
+import org.dspace.content.crosswalk.MetadataValidationException;
+import org.dspace.content.factory.ContentServiceFactory;
+import org.dspace.content.packager.METSManifest;
+import org.dspace.content.service.CollectionService;
+import org.dspace.content.service.CommunityService;
+import org.dspace.content.service.InstallItemService;
+import org.dspace.content.service.ItemService;
+import org.dspace.content.service.WorkspaceItemService;
+import org.dspace.services.ConfigurationService;
+import org.dspace.services.factory.DSpaceServicesFactory;
+import org.jdom.Element;
+import org.junit.Test;
+
+/**
+ * Basic integration testing for the Packager restore feature
+ *
+ * @author Nathan Buckingham
+ */
+public class PackagerIT extends AbstractIntegrationTestWithDatabase {
+
+    private ItemService itemService = ContentServiceFactory.getInstance().getItemService();
+    private CollectionService collectionService = ContentServiceFactory.getInstance().getCollectionService();
+    private CommunityService communityService = ContentServiceFactory.getInstance().getCommunityService();
+    private WorkspaceItemService workspaceItemService = ContentServiceFactory.getInstance().getWorkspaceItemService();
+    protected static final InstallItemService installItemService = ContentServiceFactory.getInstance()
+            .getInstallItemService();
+    protected ConfigurationService configService = DSpaceServicesFactory.getInstance().getConfigurationService();
+    protected Community child1;
+    protected Collection col1;
+    protected Item article;
+    File tempFile;
+
+    @Test
+    public void packagerExportUUIDTest() throws Exception {
+        context.turnOffAuthorisationSystem();
+        createTemplate();
```

Comment:
@ConfusionOrb221 : I *think* your random failures are being caused by not cleaning up the content you've created in your tests.  Several of your tests call `createTemplate()` to create several objects in the database...but, it appears to me those created objects are NEVER removed.  

Allowing objects to stay in the test database after tests complete is bad practice, as it can result in later ITs failing over unexpected content in the test database.

So, I think you may need want to refactor this set of tests to have a method like this:
```
@Before
public void setup() {
   // You could also just move the content of createTemplate() and createFiles() into this method
   createTemplate();  
   createFiles();
}
```

Then, have a corresponding `@After` method:
```
@After
public void destroy() {
    deleteTemplate(); //doesn't exist yet
    deleteFiles();  // doesn't exist yet
}
```

That'd ensure the same content is created before each `@Test` and then deleted after each test completes.

Reply:
Did the above changed and it did fix that test case that we were running into seemed to be that the handle was running into something I had created previously.